### PR TITLE
ci: add a workflow for the release pipeline

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: .github/workflows/releaseinfo.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            }
+            const asset = assets[0];
+            core.setOutput('url', asset.url);
+      - run: |
+          curl -L -o index.yaml -H 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' -H 'Accept: application/octet-stream' ${{ steps.get_asset_url.outputs.url }}
+      - uses: EndBug/add-and-commit@v6
+        with:
+          add: '["_data/releaseinfo.json", "index.yaml"]'
+          author_name: move2kube
+          author_email: move2kube@gmail.com
```
Comment (ashokponkumar) [inline .github/workflows/releaseinfo.yml:38]: Can we change it to move2kube-dev@googlegroups.com

File: .github/workflows/releaseinfo.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          repo: move2kube
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+      - run: echo '${{ steps.info.outputs.release_info }}' > _data/releaseinfo.json
+      - id: get_asset_url
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.MOVE2KUBE_PATOKEN }}
+          script: |
+            const response = await github.repos.listReleases({ owner:context.repo.owner, repo:'move2kube-operator' });
```
Comment (ashokponkumar) [inline .github/workflows/releaseinfo.yml:22]: Not for this PR, but we should move our helm chart to move2kube-ui repo

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Minor suggestions

File: .github/workflows/releaseinfo.yml (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply .github/workflows/releaseinfo.yml:22]: ok

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**